homeworks/hw11/bank_deposit/bank.py

```diff
@@ -0,0 +1,46 @@
+class Bank:
+    def __init__(self):
+        self.clients = {}
+        self.accounts = {}
+
+    def register_client(self, client_id, name):
+        if client_id in self.clients:
+            return False
+        self.clients[client_id] = {"name": name}
```

Comment:
Not sure if you need to have dict here, maybe it's a good idea to simplify it?
```suggestion
        self.clients[client_id] = name
```